Automattic/wp-calypso
# Update Our Color Palette to the Most Recent Version

![Comparison](https://user-images.githubusercontent.com/2060580/80515432-41bed280-8982-11ea-9432-811b78f99f13.gif)
#### Changes proposed in this Pull Request

* Upgrade `@automattic/color-studio` to the most recent version and update the default color scheme.
* Update the SVG audit script and all affected images with the new color values.

#### Testing instructions

* Pull the changes:
    * `git checkout update/palette`
    * `yarn`
* Run the SVG color audit script and make sure it doesn’t report any specific paths.
    * `node bin/audit-svg-colors.js`
* Preview the images listed by the command below (previewing locally should be enough) and make sure none of them throws any errors that would prevent them from displaying.
    * `git show HEAD~1 --name-only | grep 'svg$'`
* `yarn start`
* Ensure the new colors have been picked by Calypso:
    * Open the documentation page for the Badge component.
    * Compare the [local version](http://calypso.localhost:3000/devdocs/design/badge) with the [production](https://wpcalypso.wordpress.com/devdocs/design/badge).
    * There should be a subtle difference between the yellow bits (the warning badge and the color scheme thumbnails).
